yt_dlp/extractor/tiktok.py

```diff
@@ -274,15 +277,29 @@ def extract_addr(addr, add_meta={}):
             'description': aweme_detail.get('desc'),
             'view_count': int_or_none(stats_info.get('play_count')),
             'like_count': int_or_none(stats_info.get('digg_count')),
-            'repost_count': int_or_none(stats_info.get('share_count')),
+            'share_count': int_or_none(stats_info.get('share_count')),
             'comment_count': int_or_none(stats_info.get('comment_count')),
+            'download_count': int_or_none(stats_info.get('download_count')),
+            'repost_count': int_or_none(stats_info.get('forward_count')),
+            'save_count': int_or_none(stats_info.get('collect_count')),
+            'whatsapp_share_count': int_or_none(stats_info.get('whatsapp_share_count')),
             'uploader': str_or_none(author_info.get('unique_id')),
             'creator': str_or_none(author_info.get('nickname')),
             'uploader_id': str_or_none(author_info.get('uid')),
             'uploader_url': user_url,
             'track': music_track,
             'album': str_or_none(music_info.get('album')) or None,
             'artist': music_author or None,
+            'origin_comment': str_or_none(aweme_detail.get('origin_comment_ids')),
+            'is_advertisement': aweme_detail.get('is_ads'),
+            'audio_start_time': aweme_detail.get('music_begin_time_in_ms'),
```

Comment:
Ditto, especially since this doesn't really mean anything without the sound info